source/SpinalHDL/Data types/Int.rst

```diff
@@ -206,7 +208,7 @@ Arithmetic
      - Subtraction with carry
      - T(max(w(x), w(y)) + 1 bits)
    * - x -| y
-     - Subtraction by sat carry bit
+     - Subtraction by saturation carry bit
```

Comment:
Can this be both MIN and MAX ?  or does this always saturate to MIN ?   If so, what is that based on the +ve or -ve of the addend ?  Does this mean the same is true for the Addition ?  Trying understand the difference in description between ADD and SUBTRACT.

Reply:
Didn't think of the case that a negative value is added to an SInt, so addition can also be either min or max (but only for SInt). Instead of overflowing it saturates -> I'd vote for keeping the simple wording and maybe add a link to https://en.wikipedia.org/wiki/Saturation_arithmetic

Reply:
I don't mind the extra info, just asking the obvious question when someone reads it.

But if I were to keep it simple (I assume you mean by that a summary description of operation, without the minValue/maxValue details of the implementation) :

`Subtraction of subtrahend with ::ref:<saturation https://en.wikipedia.org/....>`.

The 'carry bit' is not a thing ?   The 2 words 'carry bit' seem out of place to me, if there is no extra 'carry bit' information that is an input or output.

Reply:
I guess the carry bit was mentioned because the HW generated does the arithmetic operation with operands extended and uses the carry bit to mux between the result or the max value.